h/static/scripts/directive/test/annotation-test.js

```diff
@@ -470,6 +470,177 @@ describe('annotation.js', function() {
       sandbox.restore();
     });
 
+    describe('AnnotationController() initialization', function() {
+      it('saves new highlights to the server on initialization', function() {
+        // New highlights have no id and have $highlight: true.
+        annotation.id = null;
+        annotation.$highlight = true;
+
+        // The user is logged-in.
+        annotation.user = fakeSession.state.userid = 'acct:bill@localhost';
+
+        annotation.$create = sandbox.stub().returns({
+          then: function() {}
+        });
+
+        createDirective();
+
+        assert.called(annotation.$create);
+      });
+
+      it('saves new highlights to drafts if not logged in', function() {
+        // New highlights have no id and have $highlight: true.
+        annotation.id = null;
+        annotation.$highlight = true;
+
+        // The user is not logged-in.
+        annotation.user = fakeSession.state.userid = undefined;
+
+        annotation.$create = sandbox.stub().returns({
+          then: function() {}
+        });
+
+        createDirective();
+
+        assert.notCalled(annotation.$create);
+        assert.called(fakeDrafts.update);
+      });
+
+      it('does not save new annotations on initialization', function() {
+        // New annotations have no id and no $highlight.
+        annotation.id = null;
+        annotation.$highlight = undefined;
+
+        annotation.$create = sandbox.stub().returns({
+          then: function() {}
+        });
+
+        createDirective();
+
+        assert.notCalled(annotation.$create);
+      });
+
+      it('does not save old highlights on initialization', function() {
```

Comment:
I'd suggest we call these "existing" rather than "old" for consistency with comments elsewhere